### PR TITLE
Adding Groups To Meetups Page

### PR DESCRIPTION
**Pull request by Atremblay9**
## What issue is this referencing?

#393

Here is a screenshot! 
image

## Do these code changes work locally and have you tested that they fix the issue yourself?

-   [x] yes!

## Does the following command run without warnings or errors?

-   [x] yes! `npm run pr-checks`

## Have you taken a look at our [contributing guidelines](https://github.com/devedmonton/DES-Website/blob/main/.github/CONTRIBUTING.md)?

-   [x] yes!

## My node version matches the one suggested when running `nvm use`?

-   [x] yes!

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -131,4 +131,6 @@ export default defineNuxtConfig({
       ],
     },
   },
-})
+
+  compatibilityDate: '2024-11-01',
```
Comment (dgmouris) [inline nuxt.config.ts:135]: Please remove!

**Comment by dgmouris** [comment]
This looks good and works for now but there are a few issues that @Atremblay9 has brought up in an in-person conversation.

- Having no description looks weird.
- As well I'm sorry to bring this up but the images are _slightly misaligned_ once you see it you can't unsee it. So maybe we could fix it some how.

I'm going to merge because it looks good to me (LGTM).

File: pages/meetups.vue [hunk 2]
```diff
@@ -20,5 +22,6 @@ defineOgImage({
 <template>
   <main>
     <AppSection :group="group" />
+    <AppSection :group="links" />
```
Comment (dgmouris) [inline pages/meetups.vue:25]: This looks good but we're going to open issue later!

**Review by dgmouris — APPROVED** [review]

**Comment by arashsheyda** [comment]
@allcontributors add @Atremblay9  for code

**Pull request merged**